xpdacq/beamtime.py

```diff
@@ -371,6 +403,18 @@ def tseries(dets, exposure, delay, num):
                         'sp_plan_name': 'tseries'})
     plan = bp.count([area_det], num, delay, md=_md)
     plan = bpp.subs_wrapper(plan, LiveTable([]))
+    def inner_shutter_control(msg):
+        if msg.command=='trigger':
```

Comment:
I think this needs spaces around the equality.

Reply:
yeah, thanks!